PendingReleaseNotes

```diff
@@ -100,6 +100,13 @@
    `ceph config get mgr mgr/volumes/snapshot_clone_no_wait`
   and it can be disabled by using:
    `ceph config set mgr mgr/volumes/snapshot_clone_no_wait false`
+* CephFS: fixes to the implementation of the ``root_squash`` mechanism enabled
+  via cephx ``mds`` caps on a client credential require a new client feature
+  bit, ``client_mds_auth_caps``. Clients using credentials with ``root_squash``
+  without this feature will trigger the MDS to raise a HEALTH_ERR on the
```

Comment:
s/raise a/raise/

Reply:
@anthonyeleven what exactly do you want to change here?

Reply:
Removing " a " on line 106.